src/Tasks/ResolveSDKReference.cs

```diff
@@ -25,6 +25,14 @@ public class ResolveSDKReference : TaskExtension
     {
         #region fields
 
+        /// <summary>
+        /// Platform aliases
+        /// </summary>
+        private static Dictionary<string, string> platformAliases = new(StringComparer.OrdinalIgnoreCase)
```

Comment:
Unlikely to matter in this case but for cleanliness:

```suggestion
        private static readonly Dictionary<string, string> PlatformAliases = new(StringComparer.OrdinalIgnoreCase)
```